bundle/1.0.0-1/hive-operator.v1.0.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,620 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: OpenShift Optional
+    certified: 'false'
+    containerImage: quay.io/openshift-hive/hive:1.0.0
+    createdAt: '2020-03-23T22:36:00Z'
+    description: OpenShift cluster provisioning and management at scale.
+    support: Devan Goodwin
+  name: hive-operator.v1.0.0-1
+  namespace: hive
+spec:
+  customresourcedefinitions:
+    owned:
+    - description: Configuration for the Hive Operator
+      displayName: Hive Config
+      kind: HiveConfig
+      name: hiveconfigs.hive.openshift.io
+      version: v1
```

Comment:
If we *only* reference the HiveConfig CRD here, I have a suspicion all the other files in this PR should not be included and would not be used.

Reply:
I was going to attempt to delete them, but was thinking then i need to tweak permissions down below and that scared me off

Reply:
I believe rbac like most kube things is tolerant of types not existing yet, so if it was rbac to crds that don't exist yet that worried you, i think that's ok. I've seen RBAC let me mistakenly grant permissions to things that don't exist many times. :)

Reply:
ok i'll test with deleting them

Reply:
`MachinePool is not in the list of crds in pkg/operator/hive/hive.go, line 172`

will be fixed later